.travis.yml

```diff
@@ -4,10 +4,11 @@ android:
   components:
   - tools
   - platform-tools
-  - build-tools-23.0.3
-  - android-23
+  - build-tools-27.0.3
+  - android-22
+  - android-27
   - extra-android-m2repository
-  - sys-img-armeabi-v7a-android-15
+  - sys-img-armeabi-v7a-android-22
```

Comment:
Because Google haven't published newer emulators without Google Play Services.
I'm not sure if such emulators would allow you to go straight into test application omitting Google setup